### PR TITLE
Added feature to core control to allow multiple functions and parameters to be assigned to one control

### PR DESCRIPTION
**Pull request by dannyparsons**
An example of this new feature is implemented in the one way frequencies dialog

### CODE REVIEW

**Comment by stevekogo** [comment]
@dannyparsons Is this ready for review?

**Comment by dannyparsons** [comment]
Yes there are some substantial changes here so it would be good to have it tested well, thanks.

**Comment by lilyclements** [comment]
I have done a review of this and made some changes on a separate branch which has been merged in. @africanmathsinitiative/developers It would be good for others to review this too as there are quite big changes!

**Comment by dannyparsons** [comment]
Thanks Lily. The main thing would just be to test a few dialogs that we previously working well and check they are still working well since there have been changes to the core control code.

**Comment by dannyparsons** [comment]
I will merge this now as I need to edit some of the files changed here on another branch. But if anyone spots bugs with the core that weren't present before this was merged let me know and I can see if this is the cause.

File: instat/dlgOneWayFrequencies.vb [hunk 1]
```diff
@@ -149,13 +172,14 @@ Public Class dlgOneWayFrequencies
         End If
     End Sub
 
-    Private Sub AllControls_ControlValueChanged(ucrChangedControl As ucrCore) Handles ucrPnlFreqDisplay.ControlValueChanged
-        ChangeBaseFunction()
```
Comment (dannyparsons) [inline instat/dlgOneWayFrequencies.vb]: Because of the new code there is now no need to change the base function each time we change the buttons.

**Pull request merged**